api-functions/cms/ContentfulAboutUs.ts

```diff
@@ -0,0 +1,39 @@
+import { gql } from "graphql-request";
+import { AboutUsContent } from "../../interfaces/contentful-about";
+import { client, CMS_ERRORS } from "./ContentfulLanding";
```

Comment:
Let's move these exports to a shared file rather than importing from the landing page. Maybe something like `contentfulClient.ts`